tests/testthat/test-expr-meta.R

```diff
@@ -0,0 +1,130 @@
+test_that("meta$eq meta$ne", {
+  # three naive expression literals
+  e1 <- pl$lit(40) + 2
+  e2 <- pl$lit(42)
+  e3 <- pl$lit(40) + 2
+
+  # e1 and e3 are identical expressions
+  expect_true(e1$meta$eq(e3))
+  expect_false(e1$meta$eq(e2))
+  expect_true(e1$meta$ne(e2))
+  expect_false(e1$meta$ne(e3))
+
+  # other is wrapped in Expr
+  expect_true(e2$meta$eq(42))
+
+  # error if not wrappable
+  expect_snapshot(
+    e2$meta$eq(complex(1)),
+    error = TRUE
+  )
+})
+
+test_that("meta$pop", {
+  e1 <- pl$lit(40) + 2
+  e2 <- pl$lit(42)$sum()
+
+  pop1 <- e1$meta$pop()
+  expect_true(length(pop1) == 2L)
+  expect_true(pop1[[1]]$meta$eq(2))
+  expect_true(pop1[[2]]$meta$eq(40))
```

Comment:
@eitsupi Those fail because I don't know how to return a "proper" polars expression from a Rust function. I tried wrapping the output in `.savvy_wrap_PlRExpr` but apparently it's not enough. Could you take a look?